server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -99,7 +99,8 @@ public final DivTag render(ApplicantQuestionRendererParams params) {
     ContainerTag questionTag;
     ImmutableList<DomContent> questionTextDoms =
         TextFormatter.createLinksAndEscapeText(
-            question.getQuestionText(), TextFormatter.UrlOpenAction.NewTab);
+            !question.isOptional() ? question.getQuestionText() + " *" : question.getQuestionText(), 
```

Comment:
Oh actually can you remove the space before the asterisk? The screenshots in the admin PR weren't updated.

Reply:
Done.

Reply:
Maybe it didn't push? the space is still there

Reply:
Should be fixed now